### ISSUE TITLE
[docs] Unclear docs of binary_cross_entropy / BCELoss

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#56581]
It says: `Function that measures the Binary Cross Entropy between the target and the output.`, but in fact it the argument names are `input` and `target`. It also does not say clearly if the inputs are probabilities, logits or log_probs. One can guess from the example, but much clearer to specify this explicitly. 

A lot of other methods like `kl_div` have the same problem

![image](https://user-images.githubusercontent.com/1041752/115567598-b8ec2580-a2bb-11eb-98d8-3ef4f2e6e11a.png)

cc @brianjo @mruberry @albanD @jbschlosser

**Comment by jbschlosser** [comment]
Agreed that the docs could use some improvement. We'd accept a PR making the terminology consistent & clarifying the expected inputs.

**Comment by Shisthruna28** [comment]
I would like to contribute to this PR. If possible, can you also mention other areas where the documentation needs improvement?

**Comment by mruberry** [comment]
> I would like to contribute to this PR. If possible, can you also mention other areas where the documentation needs improvement?

Hey @Shisthruna28! Thanks for your interesting, I think starting with just these docs would be a good amount for one PR and to get started. PRs addressing multiple topics are more challenging to review
